packages/agent-utils/src/sys/Env.ts

```diff
@@ -3,13 +3,15 @@ export class Env {
   public readonly GPT_MODEL: string;
   public readonly CONTEXT_WINDOW_TOKENS: number;
   public readonly MAX_RESPONSE_TOKENS: number;
+  public readonly BRAVE_API_KEY: string | null;
 
-  constructor(envVars: Record<string, string>) {
+  constructor(envVars: Record<string, string | null>) {
```

Comment:
Env vars that are not initialized are `undefined`, not `null`. Really love JS for adding `null` as a primitive :sweat_smile: